### PR TITLE
add header and change social icons for ability change color

### PR DESCRIPTION
**Pull request by lekar89**
Add a header and change social icons for ability to change color.

### CODE REVIEW

File: src/css/header.css [hunk 1]
```diff
@@ -1,4 +1,162 @@
-.header {
-  padding: 20px 0;
+.header-section {
+  margin-bottom: 8px;
 }
 
+.header-container {
```
Comment (kharkov-timur) [inline src/css/header.css]: Please create separate rule for container in base.css file and use in this place

**Review by kharkov-timur — APPROVED** [review]

**Comment by Fijji** [comment]
Please, add your file to index.html <load="partials/header.html" />

**Comment by lekar89** [comment]
created separate rule for container in base.css file and added  file to index.html

**Comment by Fijji** [comment]
resolved merge conflict to proceed with merge

**Pull request merged**